test/games_test.rb

```diff
@@ -66,4 +66,40 @@ def test_it_can_create_a_summary_for_the_games
 
     assert_equal expected, games.summary
   end
+
```

Comment:
These tests are great! The only issue is that they need to be in a separate file called `event_test.rb`